### PR TITLE
Add NxFilter plugin

### PR DESCRIPTION
**Pull request by xTITUSMAXIMUSX**
This is my first plugin. Let me know if I need to do anything else.

### CODE REVIEW

File: nxfilter.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "pkgs": [
+        "curl",
+        "openjdk8-jre"
+        ],
+  "packagesite": "http://pkg.FreeBSD.org/${ABI}/quarterly",
+  "fingerprints": {
+                  "plugin-default": [
+                                  {
+                                  "function": "sha256",
+                                  "fingerprint": "b0170035af3acc5f3f3ae1859d>
```
Comment (fulder) [inline nxfilter.json:18]:
```suggestion
                                  "fingerprint": "b0170035af3acc5f3f3ae1859dc717101b4e6c1d0a794ad554928ca0cbb2f438"
```

File: nxfilter.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                          }
+                        ]
+        },
+        "official": "Community"
```
Comment (fulder) [inline nxfilter.json:22]: Think this should be `"official": false`

File: nxfilter.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+}
```
Comment (fulder) [inline nxfilter.json:23]: Could you format this JSON with 4 spaces? :) Seems a bit odd right now.

File: nxfilter.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  "artifact": "https://github.com/xTITUSMAXIMUSX/iocage-plugin-nxfilter.git",
+  "properties": {
+        "allow_raw_sockets":"1",
+        "boot":"on"
+          },
+  "pkgs": [
+        "curl",
+        "openjdk8-jre"
+        ],
+  "packagesite": "http://pkg.FreeBSD.org/${ABI}/quarterly",
```
Comment (fulder) [inline nxfilter.json:13]: Are you using `quarterly` for some special reason, or isn't `latest` better in most cases?

**Review by fulder — CHANGES_REQUESTED** [review]
Thank you for contributing and good job with a first plugin. I have a few small change requests.

The plugin installation tests are currently failing but seem like it's because of the JSON syntax error (see one of the review comments). I managed to install your plugin manually without any issues so should probably pass and get green after that fix.

File: nxfilter.json (hunk 4 above)
Comment (xTITUSMAXIMUSX) [reply nxfilter.json:13]: Yes. Latest would be best. I have that updated. Thanks for pointing that out.

File: nxfilter.json (hunk 3 above)
Comment (xTITUSMAXIMUSX) [reply nxfilter.json:23]: Yeah it was my first attempt. I think I got it formatted with 4 spaces now.

File: nxfilter.json (hunk 2 above)
Comment (xTITUSMAXIMUSX) [reply nxfilter.json:22]: Ok I updated that section to false

Comment (fulder) [reply nxfilter.json:22]: I can't see this update yet? :worried:

**Review by fulder — APPROVED** [review]
Looks good to me now and the tests are green. Adding my approval :)

**Review by sonicaj — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**